spec.html

```diff
@@ -44510,6 +44558,9 @@ <h1>Properties of Generator Instances</h1>
             <td>
               [[GeneratorBrand]]
             </td>
+            <td>
+              a brand
```

Comment:
Does this actually restrict this field in any way? Is it more useful to say this than "anything"?

Reply:
Within the spec, it's `a String or ~empty~`, so I'd be inclined to put that in. (Or are we allowing for the possibility of spec-external uses that need a wider type?)

Reply:
`a String or ~empty~` works for me.